modules/nw-ovn-ipsec-north-south-enable.adoc

```diff
@@ -18,16 +18,82 @@ After you apply the machine config, the Machine Config Operator reboots affected
 .Prerequisites
 
 * Install the {oc-first}.
+* You have installed the `butane` utility on your local computer.
+* You have installed the NMState Operator on the cluster.
 * You are logged in to the cluster as a user with `cluster-admin` privileges.
 * You have reduced the size of your cluster MTU by `46` bytes to allow for the overhead of the IPsec ESP header.
-* You have installed the `butane` utility.
 * You have an existing PKCS#12 certificate for the IPsec endpoint and a CA cert in PEM format.
 
 .Procedure
 
 As a cluster administrator, you can enable IPsec support for external IPsec endpoints.
 
-. Create an IPsec configuration file named `ipsec-endpoint-config.conf`. The configuration is consumed in the next step. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/securing_networks/configuring-a-vpn-with-ipsec_securing-networks#configuring-a-vpn-with-ipsec_securing-networks[Libreswan as an IPsec VPN implementation].
+. To enable IPsec encryption, enter the following command:
++
+[source,terminal]
+----
+$ oc patch networks.operator.openshift.io cluster --type=merge \
+-p '{
+  "spec":{
+    "defaultNetwork":{
+      "ovnKubernetesConfig":{
+        "ipsecConfig":{
+          "mode":<mode>
+        }}}}}'
+----
++
+where:
++
+--
+`mode`:: The IPsec behavior. Specify `External` to encrypt only traffic to external hosts or specify `Full` to encrypt pod to pod traffic and traffic to external hosts. By default, IPsec is disabled.
```

Comment:
Traffic to external hosts is optional, mode `Full` is used also for EW ipsec without NS IPsec